samcli/lib/telemetry/event.py

```diff
@@ -0,0 +1,73 @@
+"""
+Represents Events and their values.
+"""
+
+from abc import ABC
+from enum import Enum
+from typing import List
+
+
+class EventName(Enum):
+    """Enum for the names of available events to track."""
+
+    USED_FEATURE = "UsedFeature"
+    DEPLOY = "Deploy"
+    BUILD_RUNTIME = "BuildRuntime"
+
+
+class EventType(ABC):
```

Comment:
As discussed previously, we probably don't need this class to be an ABC?

Reply:
Right, almost forgot about that 😅 will remove and fix for the next commit